Source/Tests/LinuxSyscalls/SignalDelegator.h

```diff
@@ -103,7 +103,7 @@ namespace FEX::HLE {
       DefaultBehaviour DefaultBehaviour {DEFAULT_TERM};
     };
 
-    std::array<SignalHandler, MAX_SIGNALS + 1> HostHandlers{};
+    std::array<SignalHandler, MAX_SIGNALS> HostHandlers{};
```

Comment:
Could you add a docstring indicating that the indexes into this array are POSIX signals offset by -1?